packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -148,7 +148,8 @@ export function DevtoolsView(): React.ReactElement {
 	const [supportedFeatures, setSupportedFeatures] = React.useState<
 		DevtoolsFeatureFlags | undefined
 	>();
-
+	const [queryTimedOut, setQueryTimedOut] = React.useState(false);
+	const queryTimeout = 30_000; // 30 seconds
```

Comment:
Nit: I always like to encourage adding the units to relevant variable names. I would suggest `queryTimeoutInMilliseconds` (even though it's a bit bulky). Helps readers be confident that they have the right units without having to navigate code too much.

Reply:
Make sense. Updated